go-controller/pkg/clustermanager/network_cluster_controller.go

```diff
@@ -190,10 +192,10 @@ func (ncc *networkClusterController) handleAddUpdateNodeEvent(node *corev1.Node)
 		return nil
 	}
 
-	return ncc.syncNodeClusterSubnet(node)
+	return ncc.syncNodeNetworkAnnotations(node)
 }
 
-func (ncc *networkClusterController) syncNodeClusterSubnet(node *corev1.Node) error {
+func (ncc *networkClusterController) syncNodeNetworkAnnotations(node *corev1.Node) error {
```

Comment:
This function change doesn't make much sense to me. Can you add some comments as to what you are trying to do here? Are you trying to leverage this function to also update an annotation with the network id?